src/test/java/com/znsio/teswiz/steps/JioCinemaSteps.java

```diff
@@ -40,4 +40,9 @@ public void iSwipeLeftOnTrendingInIndiaSection() {
     public void iAmAbleToViewNumberTrendingCinema(int movieNumberOnScreen) {
         new JioCinemaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).verifyMovieNumberVisibleOnScreen(movieNumberOnScreen);
     }
+
+    @When("I swipe {string} trending no {int} on trending in india section")
+    public void iSwipeRightTrendingNoOnTrendingInIndiaSection(String direction, int movieNumberOnScreen) {
+        new JioCinemaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).swipeMovieTrendingInIndiaSection(direction,movieNumberOnScreen);
```

Comment:
instead of passing the direction as a string, convert it to the Direction enum here itself, and pass that as a parameter - Direction.valueOf(direction.toUpperCase()
See this implementation in AutoScrollSteps.java - iShouldBeAbleToScrollInDynamicLayer

Reply:
Updated with Direction object in method parameters